authors/apps/articles/views.py

```diff
@@ -10,13 +10,18 @@
 from authors.apps.articles.serializers import ArticleSerializer, RatingsSerializer
 from authors.apps.articles.permissions import IsOwnerOrReadOnly
 from authors.apps.articles.renderers import ArticleJSONRenderer, RatingJSONRenderer
-from authors.apps.articles.models import Articles, Ratings
+from authors.apps.articles.models import Articles, Ratings, Favorite
 
 class ArticleNotFound(Exception):
     pass
 
 class RatingNotFound(Exception):
     pass
+    
+from authors.apps.articles.renderers import ArticleJSONRenderer
+from authors.apps.articles.models import Articles, Favorite
+
+from .serializers import FavoriteSerializer
```

Comment:
Group all imports together, ie. this should be grouped with the Ines above

Reply:
This came about when rebasing. Noted.